src/config.c

```diff
@@ -499,6 +499,9 @@ void config_load(void) {
 #ifdef __SWITCH__
 	config_set_int(CONFIG_GAMEPAD_ENABLED, true);
 	config_set_str(CONFIG_GAMEPAD_DEVICE, "any");
+	config_set_int(CONFIG_FULLSCREEN, 0);
+	config_set_int(CONFIG_VID_WIDTH, nxGetInitialScreenWidth());
+	config_set_int(CONFIG_VID_HEIGHT, nxGetInitialScreenHeight());
```

Comment:
Not sure what this is for. If SDL is supposed to automatically resize the window based on the docked/undocked state, it should enforce the correct size when the window is created too. Does it not do that?

Reply:
Yeah it doesn't, because Switch SDL port only handles the resize after the proper event, and not initially, but that's probably irrelevant since the width/height present in the config is used to create the window anyway, which would be whatever resolution was the native when Taisei was last launched

Reply:
This falls under the same category as the `SDL_WINDOWEVENT_SIZE_CHANGED` thing for me… I'm pretty unhappy about this strange implementation choice in the SDL port and would consider it a bug, but I'm ok with keeping this workaround until it's fixed, if ever.